app/src/androidTest/java/com/nervousfish/nervousfish/test/ExampleUnitTest.java

```diff
@@ -16,8 +20,9 @@
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 @SuppressWarnings("PMD")
+@RunWith(AndroidJUnit4.class)
```

Comment:
Is this needed given [line 75](https://github.com/ericcornelissen/NervousFish/blob/99a350269917d364076cb370f4be44c730d66075/app/build.gradle#L75) of build.gradle?